Makefile

```
@@ -37,7 +37,7 @@ define ver_date
 	-e "s/{{YEAR}}/${YEAR}/" $(1) || $(SED) -i -e "s/{{VER}}/DEV/g" -e "s/{{DATE}}/$(DATE)/g" $(1)
 endef
 
-ALL: Makefile package.json .$(VERSION) assets/classDiagram.svg dist/base.js dist/lips.js dist/lips.esm.js dist/lips.min.js dist/lips.esm.min.js README.md dist/std.min.scm dist/std.xcb docs/reference.json docs/version.json
+ALL: package.json .$(VERSION) assets/classDiagram.svg dist/base.js dist/lips.js dist/lips.esm.js dist/lips.min.js dist/lips.esm.min.js README.md REFERENCE.md dist/std.min.scm dist/std.xcb docs/reference.json docs/version.json
```

checkmake (0.3.2)

[warning] Target "ALL" should be declared PHONY.

(phonydeclared)